build.sbt

```diff
@@ -1,79 +1,75 @@
-
-val commonRootSettings = Seq(
-  organization := "me.jeffmay",
-  organizationName := "Jeff May",
-  
-  // set the scala version on the root project
-  scalaVersion := "2.11.8",
-
-  // fail the build if the coverage drops below the minimum
-  coverageFailOnMinimum := true,
-  coverageMinimum := 80
-)
-
-lazy val root = (project in file("."))
-  .settings(commonRootSettings)
-  .settings(
-    name := "root",
-    // don't publish the surrounding multi-project root
-    publish := {}
-  )
-  .aggregate(
-    `play23-core`,
-    `play25-core`
-  )
-
-val commonSettings = commonRootSettings ++ Seq(
-
-  version := "1.0.0",
-
-  scalacOptions ++= Seq(
-    "-encoding", "UTF-8",
-    "-deprecation:false",
-    "-feature",
-    "-Xfatal-warnings",
-    "-Ywarn-dead-code"
-  ),
-
-  resolvers ++= Seq(
-    "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
-  ),
-
-  ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) },
-
-  // don't publish the test code
-  publishArtifact in Test := false,
-
-  // disable compilation of ScalaDocs, since this always breaks on links
-  sources in(Compile, doc) := Seq.empty,
-
-  // disable publishing empty ScalaDocs
-  publishArtifact in (Compile, packageDoc) := false,
-
-  licenses += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
-)
-
-lazy val playVersion = settingKey[String]("The version of Play Framework")
-
-val coreCommonSettings = commonSettings ++ Seq(
-  coverageMinimum := 80,
-  libraryDependencies ++= Seq(
-    "com.typesafe.play" %% "play" % playVersion.value,
-    "com.typesafe.play" %% "play-test" % playVersion.value % "test",
-    "org.scalatest" %% "scalatest" % "3.0.0" % Test
+import Dependencies._
+
+name := "play-test-ops-root"
+
+ThisBuild / organization := "me.jeffmay"
+ThisBuild / organizationName := "Jeff May"
+
+// set the scala version on the root project
+ThisBuild / scalaVersion := Scala_2_11
+
+ThisBuild / licenses += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
+
+def commonProject(id: String, path: String): Project = {
+  Project(id, file(path)).settings(
+
+    scalacOptions ++= Seq(
+      "-encoding", "UTF-8",
+      "-deprecation:false",
+      "-feature",
+      "-Xfatal-warnings",
+      "-Ywarn-dead-code"
+    ),
+
+    // don't publish the test code
+    Test / publishArtifact := false,
+
+    // disable compilation of ScalaDocs, since this always breaks on links
+    Compile / doc / sources := Seq.empty,
+
+    // disable publishing empty ScalaDocs
+    Compile / packageDoc / publishArtifact := false
+
+  ).enablePlugins(SemVerPlugin)
+}
+
+def coreProject(includePlayVersion: String): Project = {
+  val playSuffix = includePlayVersion match {
```

Comment:
This feels like you should turn `Play_2_X` into case classes and this is one of the fields/methods on that object. (Matching on objects is often a code smell.)